components/Form/Form.styled.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+export const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+`;
+
+export const StyledLabel = styled.label`
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  font-weight: bold;
+`;
+
+export const StyledButton = styled.button`
+  margin: 0;
+  padding: 0 1rem;
+  height: 2.5rem;
+  border-radius: 0.5rem;
+  border: none;
+  box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
+  background-color: rgba(0, 0, 0, 0.1);
+`;
```

Comment:
Instead of using a button component here with the exact same styling as the StyledInput, you can use also a StyledInput as a Submit Button. Check this :) https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit 
and check the comment above in the Form.js